helm-chart/renku-gateway/values.yaml

```diff
@@ -68,7 +57,7 @@ image:
   tag: 'v2.0'
   pullPolicy: IfNotPresent
 
-  ## Define image for auth middleware
+  ## Define the image for the auth middleware
```

Comment:
I don't even have to look at the blame to know who wrote this comment! amirite @jirikuncar ?? :)